<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>652</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1021</width>
    <height>652</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1021</width>
    <height>652</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UTP Student Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/icono.jpg</normaloff>img/icono.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>651</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1021</width>
      <height>652</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ecf4f9;</string>
    </property>
    <widget class="QLabel" name="Bienvenido">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>305</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; color:#1a055d;&quot;&gt;¡Te damos la bienvenida!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="subtitulo">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>311</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 12pt &quot;Calibri&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Administra datos de estudiantes, cursos y profesores en un solo lugar de forma eficiente.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="imgLogin">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>320</y>
       <width>261</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>img/logo.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>441</width>
      <height>651</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>441</width>
      <height>651</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QLineEdit" name="txtUsuario">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>380</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	padding: 5px;
	background-color: white;
	border:  1px solid black;
	border-radius: 3px;
	height:50px
}

QLineEdit:hover{
	border-color: rgb(181, 12, 47);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Ingresa tu nombre de usuario</string>
     </property>
    </widget>
    <widget class="QLabel" name="logoUTP">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>img/icono.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPassword">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>460</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	padding: 5px;
	background-color: white;
	border:  1px solid black;
	border-radius: 3px;
	height:50px
}

QLineEdit:hover{
	border-color: rgb(181, 12, 47);
}
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Ingresa tu contraseña</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnIngresar">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>530</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border: none;
	border-radius: 5px;
	background-color: rgb(181, 12, 47);
}

QPushButton:hover{
	background-color: rgb(218, 13, 58);
}</string>
     </property>
     <property name="text">
      <string>Ingresar</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbRol">
     <property name="geometry">
      <rect>
       <x>88</x>
       <y>290</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 5px;
background-color: rgb(255, 255, 255);
border: 1px solid black;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="imgUser">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>390</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background-color: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>img/usuario.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="imgContra">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>470</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>img/contraseña.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Rol">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>260</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Rol:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Usuario">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>350</y>
       <width>66</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Usuario:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Contrasena">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>430</y>
       <width>95</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Contraseña:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Hola">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>94</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;¡Hola!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="IngresaTusDatos">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>240</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 12pt &quot;Calibri&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Ingresa tus datos para&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt; iniciar sesión&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <zorder>txtUsuario</zorder>
    <zorder>logoUTP</zorder>
    <zorder>txtPassword</zorder>
    <zorder>btnIngresar</zorder>
    <zorder>cmbRol</zorder>
    <zorder>Rol</zorder>
    <zorder>Usuario</zorder>
    <zorder>Contrasena</zorder>
    <zorder>imgContra</zorder>
    <zorder>imgUser</zorder>
    <zorder>Hola</zorder>
    <zorder>IngresaTusDatos</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
